js/templates/modals/purchase/purchase.html

```diff
@@ -97,9 +95,10 @@ <h2 class="h4 required"><%= ob.polyT('purchase.paymentTypeTitle') %></h2>
           <% } %>
         </div>
       </section>
-      <% if (hasModerators) { %>
+      <% if (ob.hasModerators) { %>
         <section class="contentBox padMd clrP clrBr clrSh3 showIfPay js-moderator">
           <h2 class="h4"><%= ob.polyT('purchase.moderatorTitle') %></h2>
+          <div class="js-moderated-errors"></div>
```

Comment:
Why was this added if nothing else was added to handle this? Is this the remnants of a refactor?

Reply:
I forgot, that was to fix #708 which is what I was originally working on when I noticed the mod section could show when there were no valid mods.